src/python/grpcio/commands.py

```diff
@@ -281,6 +282,18 @@ def get_ext_filename(self, ext_name):
         return filename
 
     def build_extensions(self):
+
+        use_short_temp = os.environ.get("GRPC_USE_SHORT_BUILD_TEMP", 0)
```

Comment:
we should use something but more discrete 


```suggestion
        use_short_temp = os.environ.get("GRPC_PYTHON_BUILD_USE_SHORT_TEMP_DIR_NAME", 0)
```